beacon_node/network/Cargo.toml

```diff
@@ -51,5 +51,5 @@ parking_lot = "0.12.0"
 environment = { path = "../../lighthouse/environment" }
 
 [features]
-# NOTE: This can be run via cargo build --bin lighthouse --features network/disable-backfill
-disable-backfill = []
+# NOTE: This can be run via cargo build --bin lighthouse --features network/disable_backfill
+disable_backfill = []
```

Comment:
I think it's standard for feature names to use `-` rather than `_`, do you mind if we `sed` this in the other direction? `s/disable_backfill/disable-backfill/g`?